plugin/core/sessions.py

```diff
@@ -1394,13 +1398,14 @@ def register_session_buffer_async(self, sb: SessionBufferProtocol) -> None:
         for data in self._registrations.values():
             data.check_applicable(sb)
         uri = sb.get_uri()
-        if uri:
+        version = sb.version
```

Comment:
Actually looks like we should be fine removing `SessionBuffer.version` and always relying on `last_synced_version` instead.